content/en/docs/faq/applications/remove-built-in-apps.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove the Built-in Applications in KubeSphere"
+keywords: "KubeSphere, OpenPitrix, Application, App"
+description: "How to remove the built-in applications in KubeSphere"
+linkTitle: "Remove Built-in Apps in KubeSphere"
+Weight: 16910
+---
+
+As an open-source and app-centric container platform, KubeSphere integrates 15 built-in apps in the App Store that is based on [OpenPitrix](https://github.com/openpitrix/openpitrix).
```

Comment:
Add the following sentence here:

They are accessible to all tenants in a workspace, while you can also remove them from the App Store. This tutorial demonstrates how to remove a built-in app from the App Store.